internal/handlers/sqlite/msg_dropindexes.go

```diff
@@ -200,7 +197,13 @@ func processDropIndexOptions(command string, v any, existing []backends.IndexInf
 				)
 			}
 
-			if index == "_id_" {
+			if len(existing) == 0 {
```

Comment:
The error message above `"BSON field 'dropIndexes.index' is the wrong type '%s', expected types '[string, object]'"` indicates that index can also be `object` not only `string`. Maybe a separate issue.

Reply:
What is your concern about? 

The error above is returned if the index from the query is not valid. It was already tested before. 
The current error is returned if the namespace doesn't exist. The "invalid index" situation gives a more prior error than a non-existent namespace, so this is fine. Or maybe I don't understand your question.

Reply:
Yeah I'm referring to wrong thing. It just appear in the diff so I'm commenting and not related to new code change so if what I'm seeing is sensible, could be another issue. In L188 we check `val.(string)` is string or not and returns error if it's other type. But I think it can be `object` because error message says that `expected types '[string, object]'`. We are not checking for `object`.

L188 snippet:
```
index, ok := val.(string)
if !ok {
	return nil, false, commonerrors.NewCommandErrorMsgWithArgument(
		commonerrors.ErrTypeMismatch,
		fmt.Sprintf(
			"BSON field 'dropIndexes.index' is the wrong type '%s', expected types '[string, object]'",
			commonparams.AliasFromType(v),
		),
		command,
	)
}
```

Reply:
Ah, got it! We do check for the object. It's just MongoDB's error message is weird and confusing, but that's how they want it to be returned :)

Look at `switch v := v.(type)` above. We check if it's a document, or an array, or a string. Then if it's an array, each element of the array should be a string. Documents within this array are not possible. See here - https://www.mongodb.com/docs/manual/reference/method/db.collection.dropIndexes/ - `indexes` is "string or document or array of strings". That's exactly what we check.

Reply:
Thanks, I was confused by the error message. I see it cannot be array of object, it has to be array of string :pray: